app/celer-g4/celer-g4.cc

```diff
@@ -151,10 +152,18 @@ void run(int argc, char** argv)
     CELER_LOG(status) << "Initializing run manager";
     run_manager->Initialize();
 
-    // Load the input file
     int num_events{0};
-    CELER_TRY_HANDLE(num_events = PrimaryGeneratorAction::NumEvents(),
-                     ExceptionConverter{"demo-geant000"});
+    if (!GlobalSetup::Instance()->GetEventFile().empty())
+    {
+        // Load the input file
+        CELER_TRY_HANDLE(num_events = HepMC3PrimaryGeneratorAction::NumEvents(),
+                         ExceptionConverter{"demo-geant000"});
```

Comment:
```suggestion
                         ExceptionConverter{"celer-g4000"});
```
and if we still have `demo-geant` elsewhere we could update those too :)